ngraph_bridge/version.cc

```diff
@@ -24,15 +24,15 @@
 // nGraph-TensorFlow bridge uses semantic versioning: see http://semver.org/
 
 #define NG_TF_MAJOR_VERSION 0
-#define NG_TF_MINOR_VERSION 19
+#define NG_TF_MINOR_VERSION 20
```

Comment:
This should be 21.0-rc0 as 20 is already on r20 branch. But we won't cur the release branch until we need to move too r22.